### ISSUE TITLE
Id6 Samsung s23 ultra connection issues 

### ISSUE THREAD

**Issue by Evanalfredd** [BimmerGestalt/AAIdrive#816]
Hello! I am having issues connecting my phone to the car, and app. I have the connected drive subscription. Everything connected except the last options which is usb not in transfer mode. But it IS in transfer mode. I attached pictures. 
I'm aware of the app protection that the s23 ultra has. I have done all the app permission settings I've seen in the other chats. 

![Screenshot_20240901_130552_AAIdrive](https://github.com/user-attachments/assets/f64af05f-a6aa-439e-b0e0-64309b951a34)

**Comment by Evanalfredd** [comment]
I would like to thank the creators as well.. lol first time user and I think this is the best thing that has happened to bmw. I just wish it would work for me. I paid $150 for the bmw connect to drive subscription just so I can take advantage of this app.. 😭

**Comment by hufman** [comment]
USB mode is flaky with MyBMW, if you want to use it you should use BMW Connected 6.4.
For Bluetooth, make sure the car shows the Apps option when pairing a new mobile device. If the Apps option is missing, then your ConnectedDrive subscription isn't active yet or maybe the functionality is under a different subscription level.

**Comment by Evanalfredd** [comment]
I tried downloading the apk file for the connect drive app 6.4. It does nothing
![Screenshot_20240901_182826_One UI Home](https://github.com/user-attachments/assets/1e818187-9657-426d-bf4a-77a327f6753f)

**Comment by hufman** [comment]
Yes that is correct. It just needs to be installed, and it will attempt any car connection in the background.
If your car supports the connection tho. Have you verified that the Apps option shows when you go to Pair a new device?

**Comment by Evanalfredd** [comment]
> Yes that is correct. It just needs to be installed, and it will attempt any car connection in the background. If your car supports the connection tho. Have you verified that the Apps option shows when you go to Pair a new device?


https://github.com/user-attachments/assets/3595c933-2105-4214-aec0-d4b546a02155

Idk if you can upload videos here. But if you can see it.. you can see the apps option is flickering.. should I uninstall the my bmw app?

Thank you for your help, I hope I get this solved 😭

**Comment by hufman** [comment]
Tha flickering is what I see when MyBMW tries to run over USB. I don't know why your MyBMW is acting weird over Bluetooth, perhaps it is confused by also having USB plugged in, and you should eliminate extra variables by trying one connection at a time.
